src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -305,6 +461,31 @@ const smart_objects::SmartObject& SetInteriorVehicleDataRequest::ControlData(
   return module_data[0];
 }
 
+void SetInteriorVehicleDataRequest::CheckAudioSource(
+    const smart_objects::SmartObject& audio_data) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  if (mobile_apis::PrimaryAudioSource::MOBILE_APP !=
+      audio_data[message_params::kSource].asInt()) {
+    if (!audio_data.keyExists(message_params::kKeepContext) ||
+        !audio_data[message_params::kKeepContext].asBool()) {
```

Comment:
rewrite this, indent level is rather unnecessary:

```
const bool should_keep_context = audio_data.keyExists(message_params::kKeepContext) && audio_data[message_params::kKeepContext].asBool();
const bool switch_source_from_app = 
      mobile_apis::PrimaryAudioSource::MOBILE_APP ==
          application_manager_.get_current_audio_source() && 
      mobile_apis::PrimaryAudioSource::MOBILE_APP !=
          audio_data[message_params::kSource].asInt();
if (!should_keep_context && switch_source_from_app) {
...
}
```

Reply:
@jacobkeeler Thanks, fixed acc4d3793afc490e51150016e0e4abcc11eeb592